CAM Platform/api-providers.md

```diff
@@ -56,14 +63,13 @@ ElasticBox-Release: 4.0
 | name | string | Required. Give the provider account a name in Cloud Application Manager. |
 | owner | string | Specify the user in Cloud Application Manager who owns the provider account. |
 
-**Amazon Web Services request parameters**
+#### Amazon Web Services request parameters
 
 |  Parameter  |      Type     |   Description   |
 |-------------|---------------|-----------------|
 | credentials | object | Required. Contains the credential object, which is either the AWS role ARN name if using Cloud Application Manager as a SaaS or the key and secret if using Cloud Application Manager as an appliance. |
 
-**Amazon Web Services request body**
-
+#### Amazon Web Services request body
```

Comment:
@gsanchezu this is missing other `credentials` types such as key+secret, specific for appliance; should this KB include them as well, or is it omitted on purpose?